src/main/java/org/fcrepo/importexport/exporter/Exporter.java

```diff
@@ -244,6 +247,8 @@ private void export(final URI uri) {
                 exportBinary(uri, describedby, external);
             } else if (linkHeaders.contains(containerURI)) {
                 exportDescription(uri);
+                // Export versions for this container
+                exportVersions(uri);
```

Comment:
No, your rationale is reasonable. I just wanted to make sure there was a reason.